### DISCUSSION TITLE
What macOS are you using?

### DISCUSSION THREAD

**Discussion by skovatch** [Ascoware/get-iplayer-automator#408] [Polls]
The latest version of Sparkle now requires macOS 10.11 or later. For a number of security-related reasons, I'd like to start using that, but GiA currently requires macOS 10.10, which is from 2014.  Ideally I would move to macOS 11 as a minimum requirement, but I've been supporting older versions for a while now. 

Since I don't use any analytics in the app I don't have usage statistics. That's why I have this poll instead. Please pick the OS version you are using to run GiA. Don't worry about . releases, just the major and minor version.

**Comment by WilliamOfGaunt** [comment]
I'm actually on 12.4, but 12.0 is close enough I guess.

**Reply by SwaleSwimmer** [reply]
Remaining on 10.14 because I have to buy a lot more software for newer versions. Plus I'm marginal on OS upgrades owing to my "old" iMac. 
I appreciate that it is a pain having to support older OSs, but I would hate to lose GIA.
Thanks for all your hard work Scott, as I've said before it's really really appreciated.

**Reply by MacMuser** [reply]
Drat, I clicked on 10.12 instead of 12  sorry

**Comment by ian02054** [comment]
10.13.6 here...

Thx

**Reply by cgareth** [reply]
Pop it on the poll, if I were you :)

**Comment by cgareth** [comment]
10.12 here, and on the poll.

Out of interest, and I know this isn't a discussion forum so please feel free to delete/mark as irrelevant, why do newer apps often have no support for older OSs? Take GiA.
In my head it's built of A and B - A works with 10.12 (for eg) and B is an add-on which works with the new-fangled 14.0 or whatever.
If 10.12 will never change, then A needn't change; B can be tinkered with as and when 14.0 is upgraded.

I know I'm wrong - could anyone tell me why?

Thanks Scott for everything.

**Comment by pebceb** [comment]
12.5
